### DISCUSSION TITLE
Unable to properly align categorized/colored blockquotes in listings

### DISCUSSION THREAD

**Discussion by tgurr** [requarks/wiki#4123] [Error / Bug Report] [answered]
When having a listing and trying to insert a blockquote it's always aligned to the left edge of the page, contrary to any text which is properly idented. When trying to manually ident the blockquote the categorization/color is lost. See:

![blockquote_02](https://user-images.githubusercontent.com/414984/123097810-c5fad180-d430-11eb-875a-ae89e4cb99ce.png)![blockquote_01](https://user-images.githubusercontent.com/414984/123097796-c2ffe100-d430-11eb-9f4c-374a1dcf8194.png)

**Comment by chancsy** [comment]
To add, the numbered lists numbering is reset after a blockquote (as shown in the picture).

**Reply by archem245** [reply]
I am having this issue as well.  Adding styling to an indented blockquote does not take effect.

**Reply by fnafox** [reply]
Try this
```md
# Nested Block Quotes
1. Info
   > Info block
   <!-- {blockquote:.is-info} -->
1. Success
   > Success block
   <!-- {blockquote:.is-success} -->
1. Error
   > Error block
   <!-- {blockquote:.is-danger} -->
1. Warning
   > Warning block
   <!-- {blockquote:.is-warning} -->
```

src: https://github.com/requarks/wiki/discussions/4732#discussioncomment-4783499

**Reply by NGPixel** [reply, marked as answer]
https://docs.requarks.io/en/editors/markdown#decorate-syntax
